package.json

```diff
@@ -57,18 +58,23 @@
     "@babel/runtime-corejs3": "^7.8.7",
     "@blockcerts/explorer-lookup": "^1.2.1",
     "@blockcerts/hashlink-verifier": "^1.3.0",
-    "@blockcerts/schemas": "^3.0.5",
+    "@blockcerts/schemas": "^3.2.1",
+    "@bloomprotocol/ecdsa-secp256k1-signature-2019": "github:lemoustachiste/ecdsa-secp256k1-signature-2019",
+    "@bloomprotocol/ecdsa-secp256k1-verification-key-2019": "github:lemoustachiste/ecdsa-secp256k1-verification-key-2019",
```

Comment:
workaround until hopefully this gets merged: https://github.com/hellobloom/ssi-sdk/pull/43

The ESM build they provide has mentions to `require` which breaks both the node build and the browser build of CVJS.